_plugins/auto-anchor.rb

```diff
@@ -21,7 +21,8 @@
         ## Remove double-quotes from titles before attempting to slugify
         title.gsub!('"', '')
         ## Use Liquid/Jekyll slugify filter to choose our id
-        id_prefix = '- {:#{{ "' + title + '" | slugify: "latin" }}} '
+        slug = "\#{{ \"#{title}\" | slugify: 'latin' }}"
+        id_prefix = "- {:#{slug} .anchor-list} <a href=\"#{slug}\" class=\"anchor-list-link\">🔗</a>"
```

Comment:
Now that I've looked at the preview for a bit of time, I don't really like covering the newsletters with the anchor glyph. I also don't think it's immediately intuitive that it's a link to that section of the document. For example, the links in the 'notable commits' section (eg https://deploy-preview-369--bitcoinops.netlify.com/en/newsletters/2019/10/16/#bitcoin-core-15437) could easily be interpreted as a link to the PR itself.

I'm leaning more towards just having these links as bullets, perhaps styled in blue as a subtle hint that they're clickable. I think that's much more pleasant to look at and doesn't display a minor feature in front of everyone the whole time:

```suggestion
        id_prefix = "- {:#{slug} .anchor-list} <a href=\"#{slug}\" class=\"anchor-list-link\">●</a>"
```

This is obviously just an aesthetic preference on my part. Let me know what you think.

Reply:
I like that idea.  I'll give it a try and see how it looks.